games-simulation/openrct2/openrct2-0.2.4.ebuild

```diff
@@ -0,0 +1,182 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils xdg-utils
+
+MY_PN="OpenRCT2"
+MY_PN_OBJ="objects"
+MY_PN_RPL="replays"
+MY_PN_TS="title-sequences"
```

Comment:
For me, it's gives a quick look in the ebuild, so I can change that fast in future. Due multiple used, I don't have to change that multiple times. About upstream, I would expect everything, since it starts with doing downloads of those files via Makefile. I would prefer to stay, but you decide.

Reply:
> For me, it's gives a quick look in the ebuild, so I can change that fast in future. Due multiple used, I don't have to change that multiple times. About upstream, I would expect everything, since it starts with doing downloads of those files via Makefile. I would prefer to stay since these are only 3 variables, not 15 ;-), but you decide.

It's only 3 in your ebuild. Keep in mind that the package manager sources many ebuilds during the build process to generate the dependency graph. Since this is not a library, the effect wouldn't be too much. However, it is important to understand how the global namespace is shared and used by all the ebuilds that participate in the build process.

For this ebuild, they may stay unless obviously if proxy-maint opposes it in the end.

Reply:
Allright. I will keep it here, since I think, it makes my work easier for that part. But I will keep in mind not to definie tons of variables. Usally I don't even have so much variables.